pkg/logs/log.go

```diff
@@ -91,6 +94,36 @@ type Logger interface {
 	Flush()
 }
 
+type FormatterFunc func(string) string
+
+type formatter interface {
```

Comment:
`FormatterFunc` and `formatter` we only need one of them. This is the most important part. The key method should looks like:
```go
func format(when, level, filepath, linenumber, msg, args...interface{})
```
Thining about this case:
1. Users want to output logs look like "INFO|2020.7.8|/path/to/my/business.go:176|Hello, here is my log"
2. Some users want to output logs look like "2020.7.8 business.go:176 [INFO] Hello, here is my log"

Taking scalability into consideration, we may define a structure:
```go
type LogRecord structure {
    when time.Time,
    level int,
    filepath string, 
    linenumber int, 
    msg string, 
    args...interface{}
}
```
The key method is changed to:
```go
func format(record LogRecord) string {}
```

So the `WriteMsgV2` is 
```go
func WriteMsgV2(record LogRecord)...
```

Reply:
Sure, different arguments for the format function would be great. I just couldn't get it to work on my own for some reason. I'm free tomorrow or on Sunday anytime from 11:00 (GMT+1) to talk about this new feature on a call and see what needs to be reworked to get a working solution

Reply:
Sorry, I need some time to prepare a demo. And because air tickets and hotels are extreme cheap due to the virus, I go vocation every weekend. I think next Monday would be better. 😄

Reply:
Sure no worries, so Monday the 7th?

Reply:
The 3rd of August. And I am free from 10:00 to 23:00(UTC+8) every weekday, so you can choose any time you like.